build/cloudbuild.npm.yaml

```diff
@@ -0,0 +1,29 @@
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This cloud build is triggered when there is code change in the wizard folder.
+# Run npm tests and lint
+
+# Run Python tests, lint, etc.
+steps:
+  - id: npm_test
+    name: npm:3.7
```

Comment:
should name be "node:16"?

You can test this by running `cloud-build-local --config=build/cloudbuild.npm.yaml --dryrun=false .`

Reply:
you're right, fixed!